### ISSUE TITLE
Segment appears on new line and does not return proper values

### ISSUE THREAD

**Issue by MathewYaldo** [TheJuki/KFormMaster#236]
## Description

There are a bunch of flaws with the Segmented element. First is a layout issue where the segment appears on the next line. Picture:
![image](https://i.imgur.com/Bjq8DLv.png)

Another big issue is that trying to get the value selected or even the id of the value selected does not work. The value is always wrong. For example, if Apples are selected then it says Banana is selected when I log print the values. In addition to that, utilizing getid() does not return the id of the selected element, but the ID of the formsegmentelement, which in my case is 5.

My code for the output:

```
//Log.wtf("X", "Gender ID is "+fruit.getId());
//Log.wtf("X", "fruit Tag is "+fruit.getTag());
//Log.wtf("X", "fruit val is "+fruit.getValue());
//Log.wtf("X", "fruit valas is "+fruit.getValueAsString());
```

This is the code I am using for generating the element:

```
        final FormSegmentedElement<ListItem> fruit= new FormSegmentedElement<>(5);
        fruit.setOptions(Arrays.asList(new ListItem(1L, "Orange"), new ListItem(2L, "Banana"), new ListItem(3L, "Apple")));
        fruit.setTitle("Select Fruit");
        fruit.setHint("Click to Select");
```

### Expected behavior

Segment should be on the same line and utilizing the get functions should return the right values.

Simply setting up the form, and creating a button that when clicked executes Log functions shows incorrect values.

To recap:
- Segment does not not go to same line.
- Attempting to use getValue or getValueAsString returns the wrong thing. (Ex: Oranges selected, but it returns Banana)
- getId() returns the tag of the element rather than the ID for the ListItem (Ex: Returns 5 for all ListItem elements when selected)

### KFormMaster version

* Version: Latest

### System information

* Android OS: 8.1/Latest
* Issue in Emulator: Untested
* Issue in physical device: Yes

**Comment by TheJuki** [comment]
I just released [8.2.1](https://github.com/TheJuki/KFormMaster/releases/latest). Please update your dependency and let me know if this is now working for you.

NOTE: I have pushed the latest version to Maven Central (mavenCentral()) and changed the group id to "com.github.thejuki".

```
implementation "com.github.thejuki:k-form-master:8.2.1"
```

**Comment by TheJuki** [comment]
Use the new segmentedInlineTitle element to display the title inline.

**Comment by TheJuki** [comment]
I am closing this issue. If you have any issues feel free to reopen.
